google_bigquery/plugin.spec.yaml

```diff
@@ -0,0 +1,96 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: google_bigquery
```

Comment:
To match the other Google Cloud plugins can we change this to `google_cloud_bigquery` and update the directory name and fix all the path references.